src/storage-control/src/generated/gapic_control/builder.rs

```diff
@@ -17,7 +17,7 @@
 pub mod storage_control {
     use crate::Result;
 
-    /// Common implementation for [super::super::client::StorageControl] request builders.
+    /// Common implementation for [crate::client::StorageControl] request builders.
```

Comment:
This is not the name of the client. It is called `crate::client::Storage`:

https://github.com/googleapis/google-cloud-rust/blob/b753d14da427d86c9603982c9cd21792381e8e82/src/storage-control/src/client.rs#L83

And if we rename the client, then the changes in src/storage-control/src/generated/gapic/builder.rs would be wrong.

You need a more sophisticated change. Maybe an option setting the client path?

Reply:
This is where we're going:

https://github.com/dbolduc/google-cloud-rust/tree/fix-veneer-builder-docs

My plan is to add an option that lets you rename a service. Then we just need to rename `StorageControl` -> `Storage`, and we are good.

Reply:
Opened #2238 from that branch (because I am sure I will delete the branch at some point). I will leave it as a draft and rebase if we go with this approach.

WDYT?